### PR TITLE
Adding galaxy.yml to build collection

### PR DESCRIPTION
**Pull request by hcherukuri**

### CODE REVIEW

**Comment by rpelisse** [comment]
@sabre1041 LGTM! Any thoughts?

File: galaxy.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+namespace: middleware_automation
+name: redhat-csp-download
```
Comment (sabre1041) [inline galaxy.yml:2]: All names need to use `_` instead of `-`

File: galaxy.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+version: 1.1
+readme: README.md
+authors:
+  - Andrew Block <ablock@redhat.com>
+  - Harsha Cherukuri <hcheruku@redhat.com>
+description: Downloads resources from the Red Hat Customer Portal.
+license_file: "LICENSE"
+tags: {}
```
Comment (sabre1041) [inline galaxy.yml:10]: Do we want to include any tags? or for a later discussion?

**Review by sabre1041 — CHANGES_REQUESTED** [review]

File: galaxy.yml (hunk 1 above)
Comment (rpelisse) [reply galaxy.yml]: @sabre1041 excellent point! @hcherukuri we should at least add redhat, rhel, rhn, subscription_manager as tags.

File: galaxy.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+namespace: middleware_automation
+name: redhat_csp_download
+version: 1.1
```
Comment (sabre1041) [inline galaxy.yml:3]: Version should be a string as well as follow semantic versioning. so `"1.1.0"`

File: galaxy.yml (hunk 2 above)
Comment (sabre1041) [reply galaxy.yml:10]: Tags also should be a list and not a dict

**Review by sabre1041 — CHANGES_REQUESTED** [review]

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**